Lib/timeit.py

```diff
@@ -328,6 +335,12 @@ def callback(number, time_taken):
 
         if verbose:
             print()
+    else:
+        if number % duplicate:
+            print("number is not divisible by duplicate.",
+                  file=sys.stderr)
+            return 2
+        number //= duplicate
```

Comment:
I'm stuck at this line. I fail to review this change, because I don't understand why you want to divide the number of loops?

I expect that --repeat 5 --duplicate 2 will run the statement 10 times and divide the timing by 10. If you want to run the statement 2 times using --duplicate 2, just don't use --repeat, no?